components/invitation_modal/overage_users_banner_notice/index.tsx

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {getCurrentUser, isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
+import {GlobalState} from 'types/store';
+import {getLicense} from 'mattermost-redux/selectors/entities/general';
+import AlertBanner from 'components/alert_banner';
+import {calculateOverageUserActivated} from 'utils/overage_team';
+import {isCurrentLicenseCloud} from 'mattermost-redux/selectors/entities/cloud';
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+import {makeGetCategory} from 'mattermost-redux/selectors/entities/preferences';
+import {PreferenceType} from '@mattermost/types/preferences';
+import {LicenseLinks, StatTypes, Preferences} from 'utils/constants';
+
+import './overage_users_banner_notice.scss';
+
+type AdminHasWatchItArgs = {
+    preferenceName: string;
+    overagePreferences: PreferenceType[];
+}
+
+const adminHasWatchIt = ({preferenceName, overagePreferences}: AdminHasWatchItArgs): boolean => {
```

Comment:
Nit: This name seems a little confusing. Mostly the `WatchIt` parts of the name. It makes it sound incredibly generic but unclear what the function is related to.

Based on other names you have in this file, it might fit more with something like `hasAdminDismissedBanner` since this preference is set when handling the banner dismissal in `handleDismiss`.